federation/client/clientset_generated/federation_clientset/typed/autoscaling/v1/autoscaling_client.go

```diff
@@ -63,26 +63,22 @@ func NewForConfigOrDie(c *rest.Config) *AutoscalingV1Client {
 
 // New creates a new AutoscalingV1Client for the given RESTClient.
 func New(c rest.Interface) *AutoscalingV1Client {
-	return &AutoscalingV1Client{c}
+	return &AutoscalingV1Client{c, scheme.ParameterCodec}
 }
 
 func setConfigDefaults(config *rest.Config) error {
-	gv, err := schema.ParseGroupVersion("autoscaling/v1")
-	if err != nil {
-		return err
+	gv := schema.GroupVersion{Group: "autoscaling", Version: "v1"}
+	if config.NegotiatedSerializer == nil {
+		return fmt.Errorf("expected non-nil NegotiatedSerializer for %v client", gv)
 	}
-	// if autoscaling/v1 is not enabled, return an error
-	if !api.Registry.IsEnabledVersion(gv) {
-		return fmt.Errorf("autoscaling/v1 is not enabled")
+	if config.ParameterCodec == nil {
```

Comment:
Would it be correct enough to auto-assign the generated one?  If this doesn't cause many problems, I don't really object, but I also don't expect many people to mess with it.